rust/ql/lib/codeql/rust/dataflow/internal/DataFlowImpl.qll

```diff
@@ -402,6 +445,37 @@ module Node {
     final override string toString() { result = PostUpdateNode.super.toString() }
   }
 
+  private class CapturePostUpdateNode extends PostUpdateNode, CaptureNode {
```

Comment:
Nit: Move this down after the class `CaptureNode`.

Reply:
What about moving `CaptureNode` up above `PostUpdateNode` instead? Having all the extensions of `PostUpdateNode` follow right after the abstract class makes it easier to see what it's domain is.